src/data/equiv/fin.lean

```diff
@@ -2,19 +2,31 @@
 Copyright (c) 2018 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Author: Kenny Lau
-
-Equivalences between finite numbers.
 -/
+
 import data.fin data.equiv.basic
 
+/-!
+# Equivalences for `fin n`
+-/
+
+universe variables u
+
 variables {m n : ℕ}
 
+/-- Equivalence between `fin 0` and `empty`. -/
 def fin_zero_equiv : fin 0 ≃ empty :=
 ⟨fin_zero_elim, empty.elim, assume a, fin_zero_elim a, assume a, empty.elim a⟩
 
+/-- Equivalence between `fin 0` and `pempty`. -/
+def fin_zero_equiv' : fin 0 ≃ pempty.{u} :=
+equiv.equiv_pempty fin.elim0
+
+/-- Equivalence between `fin 1` and `punit`. -/
```

Comment:
I wonder if all of these `equiv`s deserve to have `@[simp]` lemmas stating how (after coercing to functions) they act on elements. I guess not necessarily in this PR?

Reply:
Rather not this PR, indeed.